src/main/java/vazkii/botania/data/BlockTagProvider.java

```diff
@@ -129,8 +123,10 @@ protected void configure() {
 				ModBlocks.corporeaBrickWall, ModBlocks.corporeaCrystalCube, ModBlocks.corporeaFunnel, ModBlocks.corporeaIndex,
 				ModBlocks.corporeaInterceptor, ModBlocks.corporeaSlab, ModBlocks.corporeaStairs);
 
-//		getOrCreateTagBuilder(ModTags.Blocks.TERRAFORMABLE).addTag(Tags.Blocks.STONE).addTag(Tags.Blocks.DIRT).addTag(BlockTags.SAND)
-//				.add(Blocks.GRASS_BLOCK, Blocks.GRAVEL, Blocks.SNOW); TODO work around forge bug or wait for fix?
+		getOrCreateTagBuilder(ModTags.Blocks.TERRAFORMABLE).add(Blocks.ANDESITE, Blocks.DIORITE, Blocks.GRANITE, Blocks.INFESTED_STONE, Blocks.STONE, Blocks.POLISHED_ANDESITE, Blocks.POLISHED_DIORITE, Blocks.POLISHED_GRANITE)
+				.add(Blocks.DIRT, Blocks.GRASS_BLOCK, Blocks.COARSE_DIRT, Blocks.PODZOL, Blocks.MYCELIUM)
+				.add(Blocks.GRASS_BLOCK, Blocks.GRAVEL, Blocks.SNOW)
+				.add(Blocks.SAND, Blocks.RED_SAND); // #minecraft:sand causes an error for some reason
```

Comment:
can you explain?

Reply:
it contains all of the blocks that were previously in forge tags that don't exist in fabric

Reply:
no, the "causes an error" part. What error? Did you try fixing it?

Reply:
trying to do .addTag(BlockTags.SAND) causes the command to fail with "An unexpected error occurred trying to execute that command." It appears to fail with any block tag and I'm not sure why, so I just put the blocks in the sand tag instead

Reply:
catch the exception and log it to see what it is. we should try to do things right.